### ISSUE TITLE
Better OSM POI generation with iD presets

### ISSUE THREAD

**Issue by matthiasfeist** [osmandapp/Osmand#9003]
the javascript editor on openstreetmap.org iD has a nice way of helping users to tag newly created features.
It relies on a database of presets found here: https://github.com/openstreetmap/iD/tree/72a99673a0328ba4f44c58dd110797cc733f7007/data/presets

I want to suggest to use these presets as well in OSMAND to make it easier to report new POIs to OSM and tag them correctly and comprehensively from the get-go.

**Comment by shadow-identity** [comment]
Yeah I would like to have this feature in Osmand. Now I'm using Osmand for most things but still have to use Osm Go! to add POIs because it also has presets.

**Comment by letypequividelespoubelles** [comment]
It would be a great help for OSM beginners. I started contributing OSM thanks to OSMAnd app and it would have been easier for me to have preset depending of the created amenity. And even now, I don't know all the tags so I would create more accurate POI. 

@vshcherb Do you plan to add this to a future milestone ?  Thanks !
